client/package-lock.json

```diff
@@ -49,6 +49,7 @@
         "@testing-library/jest-dom": "^5.0.2",
         "@testing-library/react": "^16.0.1",
         "@types/js-cookie": "^3.0.6",
+        "@types/node": "^22.9.0",
```

Comment:
why do we need this change for this PR?

Reply:
I must have updated Node while trying to fix one or more problems with this. This looks familiar.
I'll remove this change.